packages/nectar/src/sagas/errors/handleErrors/handleErrors.saga.test.ts

```diff
@@ -11,40 +11,43 @@ import {
   IdentityState
 } from '../../identity/identity.slice'
 import { Identity } from '../../identity/identity.types'
+import { SocketActionTypes } from '../../socket/const/actionTypes'
 import { StoreKeys } from '../../store.keys'
 import { errorsActions } from '../errors.slice'
+import { ErrorCodes, ErrorMessages, GENERAL_ERRORS } from '../errors.types'
 import { handleErrorsSaga } from './handleErrors.saga'
 
 describe('handle errors', () => {
-  test('registrar error', async () => {
-    const community: Community = {
-      name: '',
-      id: 'id',
-      registrarUrl: 'registrarUrl',
-      CA: { rootCertString: 'certString', rootKeyString: 'keyString' },
-      rootCa: '',
-      peerList: [],
-      registrar: null,
-      onionAddress: '',
-      privateKey: '',
-      port: 0
-    }
-    const identity: Identity = {
-      id: 'id',
-      hiddenService: { onionAddress: 'onionAddress', privateKey: 'privateKey' },
-      dmKeys: { publicKey: 'publicKey', privateKey: 'privateKey' },
-      peerId: { id: 'peerId', pubKey: 'pubKey', privKey: 'privKey' },
-      zbayNickname: '',
-      userCsr: undefined,
-      userCertificate: ''
-    }
+  const community: Community = {
```

Comment:
factory-girl?

Reply:
I'll change that later after we merge everything with changing the name to Quiet https://github.com/ZbayApp/monorepo/pull/160